native~/Runtime/src/CesiumGlobeAnchorImpl.cpp

```diff
@@ -174,7 +192,8 @@ void CesiumGlobeAnchorImpl::SetLocalToEastUpNorthRotation(
   GlobeAnchor cppAnchor(
       UnityTransforms::fromUnity(anchor._localToGlobeFixedMatrix()));
 
-  LocalHorizontalCoordinateSystem eastUpNorth = createEastUpNorth(cppAnchor);
+  LocalHorizontalCoordinateSystem eastUpNorth =
+      createEastUpNorth(cppAnchor, getAnchorEllipsoid(anchor));
```

Comment:
Can we call `getAnchorEllipsoid` just once here?

I'm assuming we are expecting the same ellipsoid between the first and second call.